theurgy.lic

```diff
@@ -13,6 +13,8 @@ class TheurgyActions
   include DRCA
 
   def initialize(settings)
+    DRC.bput('stop play', /You stop/)
```

Comment:
Rather than stop playing, should this pause the `play` script instead then resume it once theurgy is done?